src/test/java/com/aws/greengrass/logmanager/model/LogFileTest.java

```diff
@@ -95,8 +123,14 @@ void GIVEN_log_file_with_more_than_target_lines_but_two_lines_WHEN_calculate_fil
         String expectedHash = calculate(builder.toString());
         builder.append(givenAStringOfSize(100));
         writeFile(file, builder.toString().getBytes(StandardCharsets.UTF_8));
-        String fileHash = file.hashString();
+        SeekableByteChannel fileByteChannel =
+                Files.newByteChannel(file.toPath(), StandardOpenOption.READ);
+        String fileHash = file.hashString(fileByteChannel);
+        String fileHash1 = file.hashString();
```

Comment:
Why do we want to keep the `file.hashString();` method around?. It would be inaccurate if the file rotates

Reply:
This is just the unit test to make sure both `hashString` can generate the same hash, which can give us more confident that they perform the same.

Reply:
But what is the use if we are not going to be using the older one?.